src/vs/workbench/contrib/terminal/browser/links/terminalWordLinkProvider.ts

```diff
@@ -68,14 +68,23 @@ export class TerminalWordLinkProvider extends TerminalBaseLinkProvider {
 			text += chars;
 		}
 
-		// Add the final link if there is one
+		// Add the final link if there is one and also remove trailing colon if exists
 		if (startX !== -1) {
-			result.push(new TerminalLink({ start: { x: startX + 1, y }, end: { x: line.length, y } }, text, this._xterm.buffer.active.viewportY, activateCallback, this._tooltipCallback, false, localize('searchWorkspace', 'Search workspace'), this._configurationService));
+			result.push(new TerminalLink({ start: { x: startX + 1, y }, end: { x: line.length, y } }, this._remove_trailing_colon(text), this._xterm.buffer.active.viewportY, activateCallback, this._tooltipCallback, false, localize('searchWorkspace', 'Search workspace'), this._configurationService));
 		}
 
 		return result;
 	}
 
+	// Remove trailing colon in the provided text
+	private _remove_trailing_colon(text: string): string {
```

Comment:
We use this naming convention: `_removeTrailingColon`

Reply:
Thanks for the comments. Have now followed the same naming convention in the latest commit :)